egs/wsj/s5/steps/diagnostic/analyze_lats.sh

```diff
@@ -50,7 +50,7 @@ $cmd JOB=1:$num_jobs $dir/log/lattice_best_path.JOB.log \
 
 $cmd JOB=1:$num_jobs $dir/log/get_lattice_stats.JOB.log \
   ali-to-phones --write-lengths=true "$model" "ark:gunzip -c $dir/ali_tmp.JOB.gz|" ark,t:- \| \
-  perl -ne 'chomp;s/(\w+)\s//;@a=split /\s;\s/, $_;$count{"begin ".$a[$0]."\n"}++;
+  perl -ne 'chomp;s/^\S+\s*//;@a=split /\s;\s/, $_;$count{"begin ".$a[$0]."\n"}++;
```

Comment:
It's strange that it's showing this line change, because this was already applied.  I just checked.  A github bug, I suppose.

Reply:
I think it's from the previous pull request.